internal/checker/terraform/rules/ali/alicloud_db_instance_rds_retention.rego

```diff
@@ -0,0 +1,55 @@
+# METADATA
+# title: "Verify RDS Instance SQL Collector Retention Period Exceeds 180 Days"
+# description: "This policy ensures that the retention period for the SQL collector in RDS instances is set to a value greater than 180 days. A longer retention period helps in preserving historical data for a more extended duration."
+# related_resources:
+# - https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/db_instance
+# custom:
+#   id: CB_TFALI_010
+#   severity: LOW
+package lib.terraform.CB_TFALI_010
+
+import future.keywords.in
+
+isvalid(block) {
+	block.Type == "resource"
+	"alicloud_db_instance" in block.Labels
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+pass[resource] {
+	some resource in input
+	isvalid(resource)
+	resource.Attributes.sql_collector_status != "Enabled"
```

Comment:
if sql_collector_status != "Enabled" this should be failed.. 
I think it is okay to remove this line